app/__init__.py

```diff
@@ -35,6 +35,11 @@ def create_app(config_name):
     login_manager.login_view = 'main.render_login'
     main_blueprint.config = application.config.copy()
 
+    @application.before_request
+    def remove_trailing_slash():
+        if request.path.endswith('/'):
+            return redirect(request.path[:-1])
```

Comment:
`redirect` defaults to 302, so we should probably add `code=301` for a permanent redirect.
Also, this will lose query parameters. I'm not sure if that's a problem, but e.g. https://preview.development.digitalmarketplace.service.gov.uk/search/?q=email ends up on `/search` with a 404 right now.